src/fsharp/vs/service.fs

```diff
@@ -754,75 +796,68 @@ type TypeCheckInfo
     //
     // The overall aim is to resolve as accurately as possible based on what we know from type inference
     
-    let GetDeclItemsForNamesAtPosition(parseResultsOpt : FSharpParseFileResults option,
```

Comment:
I cleaned up some of the code in ``service.fs`` to remove the use of (confusing) inner functions.